modules/ROOT/pages/prerequisites/prerequisites.adoc

```diff
@@ -273,6 +273,7 @@ ____
 == Filesystems and Shared Storage
 
 Infinite Scale defines drivers for filesystems to store blobs and metadata. The drivers can be configured via the xref:{s-path}/storage-users.adoc[Storage-Users Service Configuration].
+The used filesystem must fully support utf-8 filenames and must be POSIX-compliant.
```

Comment:
what happens when one decides to use unicode file names? :thinking:

Reply:
Files get lost if the storage decides to `normalize` the filenames to iso-xxxx-xx ....

Reply:
Yes